leaflet/tests/tests.py

```diff
@@ -15,6 +17,44 @@
 from ..forms import fields
 
 
+class DummyStaticFilesStorage(StaticFilesStorage):
+
+    def url(self, name):
+        raise ValueError
+
+
+class AppLoadingTest(SimpleTestCase):
+
+    def test_init_with_non_default_staticfiles_storage(self):
+        """
+        Non-default STATICFILES_STORAGE (ex. django.contrib.staticfiles.storage.ManifestStaticFilesStorage)
+        might raise ValueError when file could not be found by this storage and DEBUG is set to False.
+
+        Ensure that _normalize_plugins_config calls `static` lazily, in order to let `collectstatic` command to run.
+
+        """
+
+        try:
+            with self.settings(STATICFILES_STORAGE='leaflet.tests.tests.DummyStaticFilesStorage', STATIC_ROOT="/", DEBUG=False):
+                staticfiles_storage._setup()  # reset already initialized (and memoized) default STATICFILES_STORAGE
+
+                try:
+                    static("a")
+                    self.fail("static must raise an exception")
+                except ValueError:
+                    pass
+
+                PLUGINS.update({
+                    'a': {'css': 'a'},
+                })
+
+                PLUGINS.pop('__is_normalized__')
+                _normalize_plugins_config()
+        finally:
+            staticfiles_storage._setup()
+            _normalize_plugins_config()
```

Comment:
maybe add some comments to explain what's expected behaviour we should have here